charleshu-8/WebDevProject
# Create chat page

Created the nav bar and sidebar for the chat page for frontend ui

Grab branch by running:

```git fetch && git switch create-chat-page```

Make sure to re-run: ```pnpm install``` to make sure you download the MUI Components React library

Then run: 
```pnpm dev```

On main page click on Test Chat Page to view

Test:
- resize the viewport 
- make sure it looks like the figma and buttons are clickable
- also make sure it scales well to different computer sizes as well
